### PR TITLE
Fix/change connection url

### PR DESCRIPTION
**Pull request by hoseinsedaqat**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1 +1,2 @@
-PORT = 
\ No newline at end of file
+PORT = 3000
+URL  = `mongodb://localhost:27017/wave_coding`
```
Comment (zoheirkhonyagar) [inline .env.example]: @hsedaqat1378 please remove the values from `.env.example` just leave the env variable with `=`. like `PORT=`.

File: src/app/index.ts [hunk 2]
```diff
@@ -18,9 +23,18 @@ export const app = async (): Promise<void> => {
...
-  app.listen(port, () => {
-    console.log(`App is running on port ${port}`);
-  });
+  const options = { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: true, useCreateIndex: true };
+
+  mongoose.set('useFindAndModify', false);
+
+  mongoose.connect(url, options).then(() =>
+    app.listen(port, () =>
+      console.log(`Server running on http://localhost:${port} and database is ${url}`)
```
Comment (zoheirkhonyagar) [inline src/app/index.ts]: @hsedaqat1378 please remove `localhost` and make it dynamic some how you should read it from env file.

File: src/app/user/user.service.ts [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+
 export class UserService {
+
   private model = User;
 
-  getUser(): string {
-    return '';
+  async getUser(req: Request, res: Response): Promise<void> {
+    try {
+      const user: IUser[] = await this.model.find();
```
Comment (zoheirkhonyagar) [inline src/app/user/user.service.ts]: @hsedaqat1378 please use findOne for this method. based on the name of the method it should find and return a single `User`. if it's gonna return array of users please rename the method to `getUsers` .

File: .env.example (hunk 1 above)
Comment (hoseinsedaqat) [reply .env.example]: that be easy to change

File: src/app/index.ts (hunk 2 above)
Comment (hoseinsedaqat) [reply src/app/index.ts]: remove localhost from console.log() ? 
 console.log(`Server running on **http://localhost:**${port} and database is ${url}`)

File: src/app/user/user.service.ts (hunk 3 above)
Comment (hoseinsedaqat) [reply src/app/user/user.service.ts]: your mean this right:
this.model.findOne({$where})

Comment (zoheirkhonyagar) [reply src/app/user/user.service.ts]: @hsedaqat1378 yes, but please get rid of $where for the moment

**Review by zoheirkhonyagar — APPROVED** [review]

**Pull request merged**